packages/runtime/src/next/api-handler/handlers/proxy-preview-mode.ts

```diff
@@ -25,6 +24,39 @@ type ProxyPreviewModeHandlerArgs =
 const routeHandlerPattern = [P.instanceOf(Request), P.any, P.any] as const
 const apiRoutePattern = [P.any, P.any, P.any] as const
 
+const previewData: MakeswiftPreviewData = {
+  makeswift: true,
+  siteVersion: MakeswiftSiteVersion.Working,
+}
```

Comment:
By the way, this is eventually going to be dynamic based on request data which is why it was next to the call to `res.setPreviewData`.

Reply:
Ahh that's why, I'll push that back down and add a note.